### DISCUSSION TITLE
Q: Dark mode off

### DISCUSSION THREAD

**Discussion by githubhabIT** [walinejs/waline#2533] [Q&A] [answered]
### Describe the bug | 问题描述

How can I make dark mode not turn on?

### Website URL | 问题网站

https://xboxrussia.tilda.ws/

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

How can I make dark mode not turn on?

### Website URL | Problem website

https://xboxrussia.tilda.ws/

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment, marked as answer]
https://waline.js.org/en/reference/client/props.html#dark set `dark` option to `false`.
